launcher/sdw_updater_gui/Updater.py

```diff
@@ -0,0 +1,356 @@
+#!/usr/bin/env python3
+"""
+Utility library for checking and applying SecureDrop Workstation VM updates.
+
+This library is meant to be called by the SecureDrop launcher, which
+is opened by the user when clicking on the desktop, opening sdw-laucher.py
+from the parent directory.
+"""
+
+import logging
+import os
+import subprocess
+from datetime import datetime
+from enum import Enum
+
+FLAG_FILE_STATUS_SD_SVS = "/home/user/sdw-update-flag"
```

Comment:
nit/optional: worth dropping these sd-svs flags into `/home/user/.securedrop_client/*` to prevent a user from deleting